Use storyshots inside Jest

roonyh opened the pull request:
This facilitates to do snapshot testing of stories within jest itself.

Create a file named stories.test.js inside your `__tests__` directory. Or use a test file you already have of any name. Add,

~~~
import testStoryshots from '@kadira/storyshots';
testStoryshots();
~~~

Commit:
Use storyshots inside jest

arunoda commented on src/index.js:
```diff
@@ -0,0 +1,24 @@
+const storybook = require('@kadira/storybook').getStorybook();
+
+export default function testStorySnapshots(options) {
+  const description = options.description || 'React storybook snapshots';
```
Change this into `options.suite` and set the default as "Storybook Snapshots".

arunoda commented on src/index.js:
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import path from 'path';
+
+require(path.resolve('./.storybook/config.js'));
```
Get the file name from the user and let's invoke it for him.

arunoda commented:
We need an section on the README on how to use this correctly. Better if we can have an example directory in this repo.

Commit:
Updated README for usage inside jest

Commit:
Updated README for usage inside jest

arunoda committed:
Merge branch 'inside-jest' of github.com:kadirahq/storyshots into inside-jest

arunoda committed:
Change README.md

arunoda committed:
Remove old files.

arunoda committed:
Use standard as the linting tool.

arunoda committed:
Make the build tooling leaner.

arunoda committed:
Remove unwanted NPM scripts.

Commit:
Updated to use config path

Now the user should give the path of the location of directory containing configuration for both React and RN.

arunoda commented on src/index.js:
```diff
@@ -18,18 +18,19 @@ const isRNStorybook =
     storybook = require.requireActual('@kadira/storybook')
-    defaultStoriesPath = path.resolve('.storybook/config.js')
+    const configDirPath = options.configPath || path.resolve('.storybook')
+    configPath = `${configDirPath}/config.js`
```
Use `path.join` here.

arunoda committed:
Make sure to resolve paths correctly.

arunoda committed:
Update README.md with screenshots.

arunoda committed:
Update README.md

arunoda committed:
Add a link to the v2.x
